code/tests/generic_322.cpp

```diff
@@ -138,10 +138,28 @@ class Generic322: public BaseTestCase {
   }
 
    private:
-    const string foo_path = mnt_dir_ "/" TEST_DIR_A "/" TEST_FILE_FOO;
-    const string bar_path = mnt_dir_ "/" TEST_DIR_A "/" TEST_FILE_BAR;
-    const string file_contents = "some random file contents...";
+    string foo_path;
+    string bar_path;
+    string expected_md5sum;
```

Comment:
Did this work for you?  run() and check_test are different child processes. You can't populate the value in run and access in check_test if Iam right. You can instead create a dummy_file in setup - do the right set of operations to reflect expected behaviour. Then in check_test compare the md5 hashes of this dummy_file against the file foo being modified in run().

Reply:
Fixed.